.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-    "eslint.enable": true
+    "eslint.enable": true,
+    "typescript.tsdk": "node_modules\\typescript\\lib"
```

Comment:
merge?
i don't know this option
@snobu

Reply:
Should be fine, we always want to use the typescript version destreamer depends on rather than a global one.